agent/agent.go

```diff
@@ -513,12 +514,29 @@ func (a *Agent) killProcs(tc *taskContext, ignoreTaskGroupCheck bool) {
 				grip.Critical(msg)
 			}
 		}
-
 		if tc.logger != nil && !tc.logger.Closed() {
 			tc.logger.Task().Infof("cleaned up processes for task: %s", tc.task.ID)
 		} else {
 			grip.Infof("cleaned up processes for task: %s", tc.task.ID)
 		}
+
+		if tc.logger != nil && !tc.logger.Closed() {
+			tc.logger.Task().Info("cleaning up Docker artifacts")
+		} else {
+			grip.Info("cleaning up Docker artifacts")
+		}
+
+		ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
```

Comment:
nit: move this to a package constant somewhere so it's easier to find/change and also I think the timeout should be 30s or 1m

Reply:
done.